keyboards/xd87/readme.md

```diff
@@ -0,0 +1,22 @@
+# XD87
+
+![XD87](https://i.imgur.com/qVyOies.jpg)
+
+
+- Keyboard Maintainer: [Alexander Fougner](https://github.com/fougner)
+- Hardware Supported: XD87 PCB v1.1
```

Comment:
```suggestion
- Hardware Supported: XD87 PCB v1.1  
```

Reply:
Am I missing something here or why is there extra trailing whitespace ?

Reply:
The trailing space is to ensure that it's formatted correctly